open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/admin/VirtualizerAdminResource.java

```diff
@@ -0,0 +1,36 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.virtualdataconnector.virtualiser.admin;
+
+import org.odpi.openmetadata.adminservices.OMAGServerAdminServices;
+import org.odpi.openmetadata.adminservices.OMAGServerOperationalServices;
+import org.odpi.openmetadata.adminservices.properties.VoidResponse;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * VirtualizerAdminResource provides the spring annotations for the server-side
+ * implementation of the administrative interface for
+ * the Virtualizer
+ */
+
+@RestController
+@RequestMapping("/open-metadata/admin-services/users/{userId}/servers/{serverName}")
+public class VirtualizerAdminResource {
```

Comment:
This class should be in a new module for the virtualizer under open-metadata-implementation/governance-servers.

Reply:
Hi Mandy, the module has been added. Thanks!